README.md

```diff
@@ -97,7 +97,7 @@ Airflow is not a streaming solution, but it is often used to process real-time d
 
 Apache Airflow is tested with:
 
-|             | Main version (dev)           | Stable version (2.10.3)      |
+|             | Main version (dev)           | Stable version (2.10.4)      |
```

Comment:
NIT. Maybe we can remove 3.8 from the list below from main version ?

Reply:
Sure, I'll update it.

Reply:
Updated it, PTAL. 

Shouldn't we also update the `Stable version (2.10.4)`?

Reply:
Stable still supports 3.8